R/DiseasyActivity.R

```diff
@@ -254,15 +304,14 @@ DiseasyActivity <- R6::R6Class( # nolint: object_name_linter
         coll$push("scenario_matrix not set. Invoke change_activity() first.")
       }
       checkmate::assert_character(type, pattern = "(work)|(school)|(home)|(other)", any.missing = FALSE, add = coll)
-      checkmate::assert_numeric(risk, lower = 0, add = coll)
+      checkmate::assert_numeric(risk, lower = 0, add = coll) # TODO: upper is 1, right?
```

Comment:
No, it just have to be non negative (Using risk = 0 is technically OK but probably not so relevant ...).